salesagility/SuiteCRM
# Fix #9494 - Force displaying line breaks to textarea fields

Closes #9494

## Description
As described in the issue, line breaks aren't displayed for textarea fields in notification messages.

This PR adds the style _"style='white-space: pre-line'"_ to the <p> tags so the line breaks will be kept.

## Motivation and Context
With this change, the descriptions of the notification templates will be displayed properly

## How To Test This
1. Create a Meeting 
2. Add text in Description field that contains line breaks
3. Add a Contact to send invitation
4. Send invitation
5. Check in the received invitation email that the Description is displayed properly.

## Types of changes

### Final checklist
